examples/llm/tests/test_model.py

```diff
@@ -243,3 +244,21 @@ def test_determinism(attention_type: str, precision):
             loss_2.backward()
             optimizer_1.step()
             optimizer_2.step()
+
+
+def test_opt_wrapping():
+    conf = {
+        'name': 'hf_causal_lm',
+        'pretrained_model_name_or_path': 'facebook/opt-125m',
+        'device': 'cpu',
```

Comment:
device isn't actually used anywhere in ComposerHFCausalLM

Reply:
good point haha